### DISCUSSION TITLE
Issue with changing password for user "mumble"

### DISCUSSION THREAD

**Discussion by VanPatch** [mumble-voip/mumble-docker#46] [Q&A] [answered]
I use a Ugreen NAS and dock on it.
Told docker to get this mumble-docker.
Gave it its RAM and cores and it startet without issues.
I can connect thru the docker app on the nas to the "mumble-dock terminal via "/bin/bash" but this is where everything stops.
The user is called mumble, but i don't know the password to it, neither do i know the SU pwd.
I cant open any files since there is no editor like nano installed. And i cant install an editor due to locked-frontend.

How do i change SU? How can i edit superuserpwd for the server? Becuase i cant create permanent channels.

Its apparently debian but most of the commands dont work.

**Comment by Krzmbrzl** [comment]
Please check out the README for how to configure the server.

**Reply by VanPatch** [reply]
I did and it didnt work, because docker commands are "unknown" so is everything else. 
So i am looking at a terminal, with no editor, no chance of changing anything and no way of entering docker commands oder read any secrets or similar.

**Reply by azlux** [reply, marked as answer]
I don't know the tools used by Ugreen, but from what I can see in that video, during the creating of the container (https://youtu.be/Cl8iDCYAZL0?t=550), you need to add a storage pool to have a persistent `/data` volume (that where the sqlite is store inside the container)

**Reply by VanPatch** [reply]
This helped to locate the sqlite file which i dropped into a viewer to get the SU password at least. 
Which allows me to create channels and stuff. 
Many Thanks
